api/REST-API.md

```diff
@@ -1,191 +1,596 @@
 # REST API
 
-### Projected Time
+## Projected Time
 
 About 2.5 hrs
 
 - Lesson: 30 min
 - Independent Practice: 120 min
 
-### Prerequisites
+## Prerequisites
```

Comment:
Do you think this is still accurate?

Reply:
Definitely not. Changed lesson time to 60 minutes.

see [1c62c05](https://github.com/Techtonica/curriculum/pull/1410/commits/1c62c05a5fef0b7f4e4a3eba5ae6459a5019094c)